matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/identity/IdentityService.kt

```diff
@@ -92,9 +92,24 @@ interface IdentityService {
 
     /**
      * Search MatrixId of users providing email and phone numbers
+     * Note the the user consent has to be set to true, or it will throw a UserConsentNotProvided failure
+     * Application has to explicitly ask for the user consent.
+     * Please see https://support.google.com/googleplay/android-developer/answer/9888076?hl=en for more details.
      */
     fun lookUp(threePids: List<ThreePid>, callback: MatrixCallback<List<FoundThreePid>>): Cancelable
 
+    /**
+     * Return the current user consent
+     */
+    fun getUserConsent(): Boolean
```

Comment:
I think Boolean function names should start with a question. What do you think about `shouldRequestUserConsent`?

Reply:
I prefer to keep the symmetry with `setUserConsent()`.
Also, if we call it `shouldRequestUserConsent()` it will have to return the negation of the previous returned value.

I update the Javadoc to be clearer about what those 2 methods does.